docker-compose.override.yml

```diff
@@ -16,6 +16,7 @@ services:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=https://+:443;http://+:80
+      - ConnectionStrings__DefaultConnection=Server=host.docker.internal;Database=Units;User ID=sa;Password=password1;MultipleActiveResultSets=true;Application Name=MilitaryManager.Attacjments.API
```

Comment:
remove

Reply:
it will be broken if we will remove this connection string.